src/sonic_ax_impl/lib/quaggaclient.py

```diff
@@ -22,15 +22,17 @@ def parse_bgp_summary(summ):
         l = ls[li]
         if l.startswith('Neighbor        '):
             break
-        if l.startswith('No IPv'): # eg. No IPv6 neighbor is configured
```

Comment:
Can we define these strings as constant at beginning so it would be easier to maintain?

Reply:
I think this is a great idea. We could define a list of criteria to skip (with comments) at one place and check them in a loop here.

Reply:
In my opinion, both the strings and the functions such as startswith or endswith are important. So keep them together help maintenance.

---
In reply to: [363017378](https://github.com/Azure/sonic-snmpagent/pull/122#discussion_r363017378) [](ancestors = 363017378)

Reply:
You could define 2 lists of strings. one for starts with and one for ends with. They are still defined together. And the code to make use of these 2 lists are also together. This way, the data and handling are separate. I think it is cleaner this way.

Reply:
We can improve this in future. This is a quick bug fix for 201811 FRR special use case required by @zhenggen-xu . Since it is tested by him, let's keep the code in this PR.